Experiences/Experiences/View Controllers/ExperienceMapViewController.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  ExperienceMapViewController.swift
+//  Experiences
+//
+//  Created by Craig Swanson on 3/23/20.
+//  Copyright © 2020 craigswanson. All rights reserved.
+//
+
+import UIKit
+import MapKit
+import AVFoundation
+
+class ExperienceMapViewController: UIViewController {
+
+    private let locationManager = CLLocationManager()
+    var userLocation: CLLocationCoordinate2D?
+    var myExperience: Experience? {
+        didSet {
+            updateViews()
+        }
+    }
+    
+    @IBOutlet var mapView: MKMapView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        locationManager.requestWhenInUseAuthorization()
+        locationManager.delegate = self
+        locationManager.desiredAccuracy = kCLLocationAccuracyBest
+        centerViewOnUserLocation()
+        myExperience = setExperience()
+        
+    }
+    
+    func setExperience() -> Experience {
+        let currentLocation = (locationManager.location?.coordinate)!
+        let experienceTitle = "Dog"
+        let picture = Experience.Picture(imagePost: UIImage(named: "recordVideo")!)
+        let video = Experience.Video(videoPost: URL(string: "myURL")!)
+        let audio = Experience.Audio(audioPost: URL(string: "myotherURL")!)
+        
+        let newExperience = Experience(experienceTitle: experienceTitle, geotag: currentLocation, picture: picture, video: video, audio: audio)
+        
+        return newExperience
+    }
+    
+    func updateViews() {
+        guard let myExperience = myExperience else { return }
+        mapView.addAnnotation(myExperience)
+    }
+    
+    private func centerViewOnUserLocation() {
+        if let location = locationManager.location?.coordinate {
```

Comment:
I think you could get rid of the name: region and use an underscore. either _ = or let _ =. Just something to keep in mind.